docs/shared_parsers_catalog/sys_fs_cgroup_memory_tasks_number.rst

```diff
@@ -0,0 +1,3 @@
+.. automodule:: insights.parsers.sys_fs_cgroup_memory_tasks_number
```

Comment:
Since you're adding a datasource, could you add a doc entry to docs/custom_datasources_index.rst as well please.